From 8b5b1c8efe0ec01d1bf6888fdfd93c3205ae3d89 Mon Sep 17 00:00:00 2001
From: Stuart Archibald <redacted>
Date: Tue, 13 Mar 2018 12:51:22 +0000
Subject: [PATCH 1/3] Remove ICC specific flags

This removes flags that are specific to ICC optimisations etc.
---
 numpy/core/setup.py | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/numpy/core/setup.py b/numpy/core/setup.py
index 4601800..2034138 100644
--- a/numpy/core/setup.py
+++ b/numpy/core/setup.py
@@ -922,12 +922,13 @@ def configuration(parent_package='',top_path=None):
             join('src', 'private', 'binop_override.h')] + npymath_sources
 
     npymathc_path = join('build', 'src.%s-%s.%s' % (sysconfig.get_platform(), sys.version_info[0], sys.version_info[1]), 'numpy', 'core', 'src', 'npymath')
-    if platform.system() == "Windows":
-        eca = ['/fp:fast=2', '/Qimf-precision=high', '/Qprec-sqrt', '/Qstd=c99']
-        if sys.version_info < (3, 0):
-            eca.append('/Qprec-div')
-    else:
-        eca = ['-fp-model fast=2', '-fimf-precision=high', '-prec-sqrt']
+#    if platform.system() == "Windows":
+#        eca = ['/fp:fast=2', '/Qimf-precision=high', '/Qprec-sqrt', '/Qstd=c99']
+#        if sys.version_info < (3, 0):
+#            eca.append('/Qprec-div')
+#    else:
+#        eca = ['-fp-model fast=2', '-fimf-precision=high', '-prec-sqrt']
+    eca=[]
     config.add_library('loops',
                        sources=loops_src,
                        include_dirs=[npymathc_path],
-- 
1.8.3.1

